lib/plenario_web/controllers/api/detail_controller.ex

```diff
@@ -11,7 +11,25 @@ defmodule PlenarioWeb.Api.DetailController do
 
     def call(conn, opts) do
       meta = MetaActions.get(conn.params["slug"], with_virtual_points: true)
+
+      if meta == nil do
```

Comment:
Burying actions like this in an `if` is bad juju in Elixir. A better way to do this is

```elixir
def call(conn, opts) do
  meta = MetaActions.get(...
  do_call(meta)
end

defp do_call(nil), do: # explode here
defp do_call(%Meta{} = meta), do: # work here
```

Reply:
Yep I can clean this up